packages/react-core/src/components/TextInput/TextInput.tsx

```diff
@@ -215,6 +226,7 @@ export class TextInputBase extends React.Component<TextInputProps, TextInputStat
           type={type}
           value={this.sanitizeInputValue(value)}
           aria-invalid={props['aria-invalid'] ? props['aria-invalid'] : validated === ValidatedOptions.error}
+          {...(expandedProps?.isExpanded && { ...ariaExpandedProps })}
```

Comment:
We should be able to just spread ariaExpandedProps here without the `expandedProps.isExpanded &&...` since we're checking that when defining ariaExpandedProps